controllers/v1/entities.js

```diff
@@ -319,5 +379,51 @@ module.exports = class Entities extends Abstract {
     });
   }
 
+        /**
+    * @api {get} /kendra/api/v1/entities/childHierarchyPath/:entityId
+    * Get entities child hierarchy path.
+    * @apiVersion 1.0.0
+    * @apiGroup Entities
+    * @apiHeader {String} X-authenticated-user-token Authenticity token
+    * @apiSampleRequest /kendra/api/v1/entities/childHierarchyPath/5da829874c67d63cca1bd9d0
+    * @apiUse successBody
+    * @apiUse errorBody
+    * @apiParamExample {json} Response:
+    * {
+    "message": "Entities child hierarchy path fetched successfully",
+    "status": 200,
+    "result": [
+        "district",
+        "block",
+        "cluster",
+        "school"
+    ]
+  }
+    */
+
+   childHierarchyPath(req) {
```

Comment:
@aman-tunerlabs - Rename it to subEntityTypeList

Reply:
@aks30  Need to tell frontend if they accept the changes.